ConfigSpace/hyperparameters/categorical.pyx

```diff
@@ -0,0 +1,398 @@
+from collections import Counter
```

Comment:
How come this `categorical.pyx` doesn't require a `.pyd` like some of the others?

Reply:
The `.pxd` is necessary if we want to use a class as a parent class. Since we don't use the CategoricalHyperparameter as a parent class, we don't need this.